Add misterdeck 3d-printed handwired stream deck with potentiometers as joystick axes (#15556)
qmk/qmk_firmware: keyboards/handwired/misterdeck/readme.md

# MisterDeck

> ![MisteRdeck](https://i.imgur.com/FTPWqkX.jpeg)

Reviewer: ```suggestion
![MisteRdeck](https://i.imgur.com/FTPWqkXh.jpeg)
```

A 3D-printed streamdeck like keyboard featuring 12 keys and 4 sliders

* Keyboard Maintainer: [Chris Broekema](https://github.com/broekema)
* Hardware Supported: MisteRdeck by MisteR_ofcl 
* Hardware Availability: [Thingiverse](https://www.thingiverse.com/thing:4627779)

Make example for this keyboard (after setting up your build environment):

>     qmk compile -kb handwired/misterdeck -km default
>     qmk flash -kb handwired/misterdeck -km default
> 
> or
> 
>     make handwired/misterdeck:default:flash

Reviewer: ```suggestion
    make handwired/misterdeck:default

Flashing example for this keyboard:

    make handwired/misterdeck:default:flash
```
